save-cloud-charts/save-cloud/templates/orchestrator-service-account.yaml

```diff
@@ -9,7 +9,13 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: Role
 metadata:
   name: jobs-executor
+  {{ if .Values.agentNamespace }}
+  namespace: {{ .Values.agentNamespace }}
```

Comment:
Does this namespace mean that pods matching selector have a role in another namespace or that pods from this namespace matching selector have a role on cluster level?

Reply:
> matching selector have a role in another namespace
this one

Role and RoleBinding should be in the same namespace, since it did work previously when role and agent's pod were in the same namespace, I moved role to agent's namespace

Reply:
oh ok